src/components/base/JInputField/JInputField.js

```diff
@@ -71,6 +70,7 @@ export function JInputField({
       )}
     >
       <div
+        onClick={handleFocus(textInput)}
```

Comment:
Creates new function on every call to render. It will cause continuous re-renders, that could lead to performance issues or memory leaks. Please use class method, memoization or hook instead.